examples/create-react-app/src/i18n.ts

```diff
@@ -0,0 +1,22 @@
+import { setupI18n } from "@lingui/core";
+import { messages as messagesEn } from "./locales/en/messages"
+import { messages as messagesCs } from "./locales/cs/messages"
+
+import { en, cs } from 'make-plural/plurals'
+
+export const locales = {
+  en: "English",
+  cs: "Česky",
+};
+
+export const defaultLocale = "en";
+
+export const i18n = setupI18n();
+i18n.load({
```

Comment:
@tricoder42 can't  we load plurals on this method?

Reply:
No, this only handles messages.

Also, please don't use `setupI18n` in tutorial. That's for advanced use and it requires additional configuration in `lingui.config.js`. Simply import `i18n` from `@lingui/core`

Reply:
Great, didn't know that I'll add that to the documentation